Sources/NIO/NonBlockingFileIO.swift

```diff
@@ -214,6 +214,48 @@ public struct NonBlockingFileIO {
         }
     }
 
+    /// Write `byteCount` bytes from `buffer` to `fileHandle` in `NonBlockingFileIO`'s private thread pool which is separate from any `EventLoop` thread.
+    ///
+    /// - parameters:
+    ///   - fileHandle: The `FileHandle` to write to.
+    ///   - buffer: The `ByteBuffer` to write.
+    ///   - byteCount: The number of bytes to write from `buffer` to `fileHandle`.
+    ///   - eventLoop: The `EventLoop` to create the returned `EventLoopFuture` from.
+    /// - returns: An `EventLoopFuture` which is fulfilled if the write was successful or fails on error.
+    public func write(fileHandle: FileHandle,
+                      buffer: ByteBuffer,
+                      byteCount: Int,
```

Comment:
sorry, I'm pretty late to the party: Do we need `byteCount`? If the user wants to write less, they can just slice the ByteBuffer, right? @FranzBusch / @Lukasa

Reply:
I'm fine with that.